instrumentation/jmx-metrics/javaagent/src/main/resources/jmx/rules/camel.yaml

```diff
@@ -1,11 +1,25 @@
 ---
 rules:
+
+  # Context Level Metrics
+
   - bean: org.apache.camel:context=*,type=context,name=*
     prefix: camel.context.
     metricAttribute:
       context: param(context)
       camelVersion: beanattr(CamelVersion)
+      packageScanClassResolver: beanattr(PackageScanClassResolver)
```

Comment:
The semantic conventions for metric attributes suggest using underscores to separate individual words, so these apparently should rather be `camel_version` and `package_scan_class_resolver`. But, given that you merely copy the MBean attributes here, I'm not sure what is more user-friendly.